src/strategies/k61r.f

```diff
@@ -2,14 +2,15 @@ FUNCTION K61R(ISPICK,ITURN,K,L,R, JA)
 C BY DANNY C. CHAMPION
 C TYPED BY JM 3/27/79
       k61r=ja    ! Added 7/27/93 to report own old value
+      IF (ITURN .EQ. 1) ICOOP = 0
```

Comment:
Let's add a comment that this was a new addition

Reply:
Yeah good call.